lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -4,56 +4,72 @@ LwM2M carrier
+
 Every released version is considered for certification with carriers.
-Refer to the release notes for each version of the library to check the certification status of a particular release.
-Your final device must go through certification by Verizon Wireless.
+Refer to the :ref:`liblwm2m_carrier_changelog` to check the certification status of a particular release.
+Your final device must go through certification by the carrier.
+
+The :ref:`lwm2m_carrier` sample demonstrates how to run this library in an application.
 
 Prerequisites
 *************
-The LwM2M carrier library has dependencies on Zephyr and the nRF Connect SDK.
-Check the release notes for the library to see which version of |NCS| it is built for.
-To integrate the library in an application, the application must be based on a supported nRF Connect SDK version.
+The LwM2M carrier library has dependencies on Zephyr and the |NCS|.
+See the :ref:`liblwm2m_carrier_changelog` of the library to check which version of |NCS| it is built for.
+To integrate the library into an application, the application must be based on a supported |NCS| version.
 
 Application integration
 ***********************
-To run the library in an application, it is mandatory to implement the application with the API of the library.
-The application has to keep the following values up to date, and the library with automatically report the updated values to the relevant server:
-Available Power Sources, Power Source Voltage, Power Source Current, Battery Level, Battery Status, Memory Free, Memory Total, Error Code, Device Type.
-The application also needs to implement an event handler and error handlers as shown in the client sample.
-The bsdlib_init function is called by the library because of the need to track modem update state.
+To run the library in an application, you must implement the application with the API of the library.
+
+The following values that reflect the state of the device must be kept up to date by the application:
+
+* Available Power Sources
+* Power Source Voltage
+* Power Source Current
+* Battery Level
+* Battery Status
+* Memory Total
+* Error Code
+* Device Type
 
-Features
-********
+The library automatically reports the updated values to the relevant server.
 
+The application must also implement an event handler and error handlers, as shown in the :ref:`lwm2m_client` sample.
+The ``bsdlib_init`` function is called by the library because of the need to track modem update state.
 
 Application limitations
 ***********************
 While running this module, the application has some limitations.
-The library depends on the LTE link controller to maintain connections to carrier servers when required, and the application has to run module.
+
+The library depends on the LTE link controller to maintain connections to carrier servers when required, so the application must run this module.
```

Comment:
This is not really a limitation, but more of a dependency for the application which comes indirectly via the glue layer and not the library (and it is managed by Kconfig). I think this line could be removed.

Reply:
removed